packages/components/src/tooltip/test/index.js

```diff
@@ -276,5 +276,39 @@ describe( 'Tooltip', () => {
 			// Tooltip won't show, since the mouse has left the anchor
 			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
 		} );
+
+		it( 'should render the shortcut display text when a string is passed as the shortcut', async () => {
+			render(
+				<Tooltip text="Help text" shortcut="shortcut text">
+					<button>Hover Me!</button>
+				</Tooltip>
+			);
+
+			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
+			button.focus();
+
+			expect( screen.getByText( 'shortcut text' ) ).toBeInTheDocument();
```

Comment:
Noting that in my experience with tests, `.toBeVisible()` is always better than `.toBeInTheDocument()`, except when the element is actually not visible. So we should try to use the former when possible:

```suggestion
			expect( screen.getByText( 'shortcut text' ) ).toBeVisible();
```

Reply:
In all tests, I replace `toBeInTheDocument` with `toBeVisible`.